src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -2,189 +2,202 @@ module sonic-vlan {
 
     yang-version 1.1;
 
-	namespace "http://github.com/Azure/sonic-vlan";
-	prefix vlan;
+    namespace "http://github.com/Azure/sonic-vlan";
+    prefix vlan;
 
-	import ietf-inet-types {
-		prefix inet;
-	}
-
-	import sonic-types {
-		prefix stypes;
-		revision-date 2019-07-01;
-	}
-
-	import sonic-extension {
-		prefix ext;
-		revision-date 2019-07-01;
```

Comment:
If you want to fix the tab and space issue, I will suggest raising a PR that changes just tab to space. Send me the link I will quickly approve that.  
And then on top of that raise this PR.

Reply:
I've reverted to the usage of tabs. Let's complete this PR now, only with the required changes (for PortChannel as Vlan member)